catalog/mvcc/cataloger_list_commits_test.go

```diff
@@ -373,7 +373,7 @@ func TestCataloger_ListCommits_LineageFromChild(t *testing.T) {
 		t.Fatal("Merge Summary", diff)
 	}
 	// TODO(barak): enable test after diff between commits is supported
```

Comment:
so I think we should enable this test then? @nopcoder @johnnyaug

Reply:
These tests use the MVCC implementation. Although we may consider trying to get them to work with the new one.
@nopcoder WDYT